doc/target-local-block-pv.md

```diff
@@ -0,0 +1,78 @@
+# How to clone an image from one block raw PV to another one
```

Comment:
So I am trying to understand what is going on in this document, and I think we are missing the point a little bit. We already have a document explaining how to create a block PV, so we don't need to repeat that. I think the point we should be trying to make is the following:
1. Given a block PVC source, and an existing block PV, what do we need to put in the target block PVC yaml to make it work. Note we should not be mixing block and non block while cloning.

Reply:
so in that case, the user needs also to assume that there is another blockDevice (/dev/loop11) already defined on the host (except the clone PV with the other local path). Is that ok with you? Not to show how to define the block Device on the host?

Reply:
The manual creation of the block PVs is done for the benefit of our CI system so we know some block PVs are available. But in general we can assume that if a user wants a block PV they have some mechanism available to create it for them. Since we have already shown the user how to manually create one block PV creating another one should be obvious.

Reply:
ok

Reply:
So any reason not to remove this file, we have basically the identical example for import?

Reply:
This is the only yaml to include clone dv example with volumeMode:Block.

Reply:
Oh I was looking at the name of the file, I thought it was describing the creation of a block PV. If you could rename the file to better reflect the purpose of the file

Reply:
clone-block-datavolume.md?

Reply:
Sounds good to me, that way the name of the file explains what is in it.